src/core/frameworks/gql/queries.ts

```diff
@@ -315,7 +328,7 @@ export const GET_BORROWER_POSITIONS_QUERY = gql`
   ${ESCROW_FRAGMENT}
 
   query getBorrowerPositions($borrower: String!) {
-    lineOfCredits(where: { borrower_contains: $borrower }) {
+    lineOfCredits(where: { borrower: $borrower }) {
```

Comment:
slight tweak to the `getBorrowerPositions` query that is more semantically correct